derive_builder_core/src/initializer.rs

```diff
@@ -59,28 +59,47 @@ pub struct Initializer<'a> {
     /// they may have forgotten the conversion from `UninitializedFieldError` into their specified
     /// error type.
     pub custom_error_type_span: Option<Span>,
+    /// Method to use to to convert the builder's field to the target field
+    ///
+    /// For sub-builder fields, this will be `build` (or similar)
+    pub conversion: FieldConversion<'a>,
 }
 
 impl<'a> ToTokens for Initializer<'a> {
     fn to_tokens(&self, tokens: &mut TokenStream) {
         let struct_field = &self.field_ident;
+        let builder_field = &*struct_field;
```

Comment:
If you can't write `let builder_field = struct_field;` (which is surprising, since I'd have guessed it impls Copy), then how about this?

```suggestion
        let builder_field = &self.field_ident;
```

Reply:
Just removing the `&*` works. Not sure why it was there; perhaps some earlier version of this code in my branch needed it.